WalletWasabi/WalletWasabi.csproj

```
@@ -22,7 +22,7 @@
 
   <ItemGroup>
 	  <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.1" />
-	  <PackageReference Include="NBitcoin" Version="5.0.13" />
+	  <PackageReference Include="NBitcoin" Version="5.0.14" />
```

https://github.com/MetacoSA/NBitcoin/releases/tag/v5.0.15
```suggestion
	  <PackageReference Include="NBitcoin" Version="5.0.15" />
```